### DISCUSSION TITLE
When you try to create a transaction, it is not registered in the blockchain.

### DISCUSSION THREAD

**Discussion by AltRex-root** [1200wd/bitcoinlib#413] [Q&A] [answered]
Good time of day.
I am trying to send 0.00005 btc from a wallet created with bitcoinlib to a wallet from TrustWallet.

Initially, I tried just dry code from the documentation.

`w = Wallet("IgorWallet")
w.scan()
t = w.send_to(to_address="bc1qfuvk2pflvhz5uvfv93vgkdgy0xx0fzkxsmhrpf", amount="0.00005 BTC")
print(t.info())`

Then I tried to convert the recipient's address to an Address object, but that didn't help either.

The problem is that the hash outputs, but it is not visible anywhere on the blockchain.

**Comment by mccwdev** [comment, marked as answer]
I'm not sure what the question is, the output address is valid and has received many transactions.

You tried to send a transaction from a bitcoinlib wallet, but the transaction was not broadcasted?

Could you check your logs and share you code if the problem persists?
